CONTRIBUTING.md

```diff
@@ -502,6 +522,59 @@ $ popd
 >**Note:** The final `popd` is required to return to the top-level directory 
 >from where other build rules can be executed.
 
+## Porting
+
+Porting is the action of applying a patch set to either an older (backwards
+port, or simply "backport") or newer (forward port) branch or repository.
+Porting is done on a new PR meaning porting PRs *must* have an associated "parent" PR
+(the original bug fix or feature PR).
+
+Porting is so important that *every PR* must indicate whether it should be
```

Comment:
Ln 532: Revise sentence to start: **Every PR** must indicate.... 
Reason: Conciseness.